### PR TITLE
[Do Not Merge] Adds Atomic App lifecycle definition. Closes #290

### LINKED ISSUE #290: document the behavior of `atomicapp verbs` and the states an atomicapp may have
beautify https://etherpad.mozilla.org/QS0TSJaPhY

### PR DESCRIPTION
**Pull request by dustymabe**
Don't merge until after beta3 when this new lifecycle will take effect.

### CODE REVIEW

File: docs/atomicapp_lifecycle.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Atomicapp Lifecycle Definition
```
Comment (goern) [inline docs/atomicapp_lifecycle.md]: pls review regarding branding.. Atomic App or atomicapp

Comment (dustymabe) [inline docs/atomicapp_lifecycle.md]: which one is it? Is there a place this is all defined?

Comment (goern) [inline docs/atomicapp_lifecycle.md]: no, maybe, we have define that we call it 'Atomic App' if we are talking about the project in a more marketingish context and 'atomicapp' if it comes to source and technical stuff...

**Comment by dustymabe** [comment]
Closing in favor of #560

**Pull request closed without merging**